foundry/game.d.ts

```diff
@@ -135,7 +135,7 @@ declare const ui: {
    * Initialized in
```

Comment:
Not related to the changes made here but I think it might make sense to split the class declaration from the declaration of the variables into separate files.